std/equal.m31

```diff
@@ -375,6 +375,16 @@ let process_eta =
     | |- ?e : ?t => deep [] e t
     end
 
+let process_reducing c l =
+  let _ = match c with
+      | |- _constant _ => ()
+      | |- _atom _ => ()
+      | |- _proj _ _ => ()
+      | _ => print "only constants and atoms may have reducing information"; failure
```

Comment:
Ok, that'll be a good start. Eventually we'll want something better, I suspect (such as printing of locations of terms, etc.)